metamusic/__main__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Author : Salvador <Salvador@fedora>
+Date   : 2022-10-07
+Purpose: New program
+"""
```

Comment:
suggestion: Add the entry point for your cli application.

Take a look at this small article
https://chriswarrick.com/blog/2014/09/15/python-apps-the-right-way-entry_points-and-scripts/